mesh/ncmesh.hpp

```diff
@@ -107,7 +107,7 @@ class NCMesh
    explicit NCMesh(const Mesh *mesh);
 
    /// Load from a stream. The id header is assumed to have been read already.
-   NCMesh(std::istream &input, int version, int &curved);
+   NCMesh(std::istream &input, int version, int &curved, int &is_nc);
```

Comment:
As an aid to other developers we should probably take this opportunity to document these arguments. What values of `version` are recognized, are `curved` and `is_nc` treated as booleans?

Reply:
Added docs.